### DISCUSSION TITLE
Observation List view

### DISCUSSION THREAD

**Discussion by jencastrodoesstuff** [digidem/mapeo-desktop#586] [Ideas]
**Describe the idea**
An observation list. Just like Mm. Currently the workflow for the new report view does not include or editing observations or even knowing how many or which observations are included in the report..

**What's your role? Where and how do you currently use the application?**
Training local partners, debugging and personal use

**What is your ideal step by step workflow?**
after maps there is view tab that says "List". Click. View observation list

**Have you thought about what it might look like?**
The first version can be a list. with the same info displayed on Mm with a scroll bar and small thumbnail previews,
the next step would be a detailed viewer pain that appears when an observation is selected.
<img width="735" alt="Screen Shot 2020-10-05 at 2 20 25 PM" src="https://user-images.githubusercontent.com/33264095/95117054-ee616500-0715-11eb-9d73-9ac167ea5e81.png">

**Additional context**
editing the observation details pane would b a nice touch.
Later iterations can have other features like "sort by" and "select for report /export."

**Comment by gmaclennan** [comment]
Back in January Aldo worked on some new designs for integrating the Observations and Territory views which include more consistent UX between mobile and desktop and a list of observations:

![image](https://user-images.githubusercontent.com/290457/95327394-e5020500-089b-11eb-8886-c3c823212bb3.png)

See [our internal design docs](https://www.notion.so/digidem/MD-Updated-Map-Observations-UX-UI-1673123ee63d42448c062205803b8a72) for more details.

We should discuss whether this is needed sooner and to work on a design that will work within the current layout of the app. One option I have been thinking about is showing a table view, e.g. each observation as a row with columns for category, description, date and location where the user can click a row to open the same observation dialog that shows from other views. This would be relatively simple to integrate (just requires implementing a table view) and would not involve much extra UX work.

**Comment by okdistribute** [comment]
The new UI prototype looks great. 

We should factor in to this analysis that a table view is yet another feature to maintain long-term, whereas a list view could be re-used from the mobile app (re-use has nice maintainability and UX properties for users!)
